specification/hdinsight/data-plane/Microsoft.HDInsight/preview/2018-11-01-preview/livySpark.json

```diff
@@ -0,0 +1,1128 @@
+      ]
+    }
+  ],
+  "securityDefinitions": {
+    "azure_auth": {
+      "type": "oauth2",
+      "authorizationUrl": "https://login.microsoftonline.com/common/oauth2/authorize",
+      "flow": "implicit",
+      "description": "Azure Active Directory OAuth2 Flow",
+      "scopes": {
+        "user_impersonation": "impersonate your user account"
+      }
+    }
+  },
+  "paths": {
+    "/livy/batches": {
+      "get": {
+        "tags": [
+          "SparkBatch"
+        ],
+        "description": "List all spark batch jobs",
+        "operationId": "SparkBatch_List",
+        "x-ms-examples": {
+          "List all spark batch jobs": {
+            "$ref": "./examples/HDI_SparkBatch_List.json"
+          }
+        },
+        "consumes": [
+          "application/json"
+        ],
+        "produces": [
+          "application/json"
+        ],
+        "parameters": [
+          {
+            "name": "from",
+            "in": "query",
+            "description": "Optional param specifying which index the list should begin from.",
+            "required": false,
+            "type": "integer",
+            "format": "int32"
+          },
+          {
+            "name": "size",
+            "in": "query",
+            "description": "Optional param specifying the size of the returned list.\r\n            By default it is 20 and that is the maximum.",
+            "required": false,
+            "type": "integer",
+            "format": "int32"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "Success",
+            "schema": {
+              "$ref": "#/definitions/SparkBatchJobCollection"
+            }
+          }
+        }
+      },
+      "post": {
+        "tags": [
+          "SparkBatch"
+        ],
+        "description": "Create a new spark batch job.",
+        "operationId": "SparkBatch_Create",
+        "x-ms-examples": {
+          "Create a spark batch job": {
+            "$ref": "./examples/HDI_SparkBatch_Create.json"
+          }
+        },
+        "consumes": [
+          "application/json"
+        ],
+        "produces": [
+          "application/json"
+        ],
+        "parameters": [
+          {
+            "$ref": "#/parameters/requestedByParameter"
+          },
+          {
+            "name": "sparkBatchJobRequest",
+            "in": "body",
+            "description": "Livy compatible batch job request payload.",
+            "required": true,
+            "schema": {
+              "$ref": "#/definitions/SparkBatchJobRequest"
+            }
+          }
+        ],
+        "responses": {
+          "201": {
+            "description": "Created",
+            "schema": {
+              "$ref": "#/definitions/SparkBatchJob"
+            }
+          }
+        }
+      }
+    },
+    "/livy/batches/{batchId}": {
+      "get": {
+        "tags": [
+          "SparkBatch"
+        ],
+        "description": "Gets a single spark batch job.",
```

Comment:
>. [](start = 53, length = 1)

From my side it will weird that it ends with `.`. I have checked x-ms-examples in cluster.json all the "key" in x-ms-examples doesn't end with `.`.  You can also check other service json and keep the style same with others.